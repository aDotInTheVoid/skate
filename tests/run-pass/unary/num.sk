fn main() {
    print -1;
    print 8----1;

    print -1.;
    print --7.--5.
}
