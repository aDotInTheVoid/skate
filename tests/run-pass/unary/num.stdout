-1
9
-1
12
